apps/noaabuoy/noaa_buoy.star

```diff
@@ -0,0 +1,147 @@
+"""
+Applet: NOAA Buoy
+Summary: Display buoy wave data
+Description: Display swell data for user specified buoy. Find buoy_id's here : https://www.ndbc.noaa.gov/obs.shtml Buoy must have height,period,direction to display correctly
+Author: tavdog
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+load("encoding/json.star", "json")
+load("cache.star", "cache")
+
+def fetch_data(buoy_id):
+    url = "http://wildc.net/wind/noaa_buoy_api.pl?buoy_id=%s" % buoy_id
+    resp = http.get(url)
+    if resp.status_code != 200:
+        fail("request failed with status %d", resp.status_code)
+        return None
+    else:
+        return resp.json()
+
+def main(config):
+    # color based on swell size
+    color_small = "#00AAFF"  #blue
+    color_medium = "#AAEEDD"  #??
+    color_big = "#00FF00"  #green
+    color_huge = "#FF0000"  # red
+    swell_color = color_medium
+
+    buoy1_id = config.get("buoy1_id", 51211)
```

Comment:
`buoy1_id` will come as a string - so you will need to type case it. I can also add support for `config.int()` but it doesn't exist today